bytecomp/meta.mli

```diff
@@ -20,7 +20,7 @@ external realloc_global_data : int -> unit = "caml_realloc_global"
 type closure = unit -> Obj.t
 type bytecode
 external reify_bytecode :
-  bytes array -> Instruct.debug_event list array -> string option ->
+  bytes array -> 'a -> string option ->
```

Comment:
I think you should try again to use an abstract type and resolve the issue in toplevel.ml using the new`Type.Equal`